app/ui/omni-box.js

```diff
@@ -107,6 +109,34 @@ class OmniBox extends HTMLElement {
     this.forwardButton.addEventListener('click', () => {
       this.dispatchEvent(new CustomEvent('forward'))
     })
+
+    // middle mouse click paste&go
+    const win = new BrowserWindow()
+    this.input.addEventListener('paste-and-go', () => {
+      win.webContents.executeJavaScript('window.location.href = ""')
+      const rawURL = clipboard.readText()
+      let url = rawURL
+
+      if (!isURL(rawURL)) {
```

Comment:
Instead of this check, we should only trigger this behavior if the pasted data *does* look like a URL.

Something like this:

```javascript
if(!isURL(url)) {
  return
} else {
  event.preventDefault()
  this.dispatchEvent(new CustomEvent('navigate', { detail: { url 
}
```

Reply:
Yeh that's right. I'll do that